cinema/models.py

```diff
@@ -0,0 +1,7 @@
+from django.db import models
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=100)
+    description = models.TextField()
```

Comment:
If you know the maximum length of the text in the `description` field, consider using `CharField` with a `max_length` parameter instead of `TextField`. This would be more efficient for your database.